lib/jake/context.ex

```diff
@@ -1,3 +1,7 @@
 defmodule Jake.Context do
-  defstruct root: %{}, child: %{}, size: 0
+  defstruct root: %{},
+            child: %{},
+            size: 0,
+            default_path: File.cwd!(),
```

Comment:
probably you could call it root_directory or something. btw, calling File.cwd! will capture the directory structure of where the code is compiled, we should probably do it when the function is called

Reply:
`root_dir` will be better and temporarily we will be set it to '/'. We can later on change it depending on project requirement, and File.cwd! will be moved to expand_ref.